extensions/sherlock-rxjs/package.json

```diff
@@ -39,6 +39,6 @@
     },
     "peerDependencies": {
         "@politie/sherlock": "0.0.0-PLACEHOLDER",
-        "rxjs": "^5.4.0 || ^6.0.0"
+        "rxjs": "^5.4.0 || ^6.0.0 || ^7.0.0 "
```

Comment:
We could consider removing ^5.4.0 as installing that in the library breaks the test (there are unknown operators used in the test).

Reply:
We could, but we can do that another time